Resources/Prototypes/Roles/Jobs/Fun/scurret_startinggear.yml

```diff
@@ -16,6 +16,12 @@
     head: ClothingHeadHatHopcap
     id: ScurretPDA
 
+- type: startingGear
+  id: ScurretMouseGear
+  equipment:
+    head: ClothingHeadHatBeretFrench
```

Comment:
we don't have a mouse hat smh

Reply:
I realised it was a silly request anyway because no one will be developing on the april fools branch... Thanks for trying anyway